src/engine/cachingreader/cachingreaderchunk.cpp

```diff
@@ -49,34 +48,51 @@ mixxx::IndexRange CachingReaderChunk::bufferSampleFrames(
         mixxx::SampleBuffer::WritableSlice tempOutputBuffer) {
     DEBUG_ASSERT(m_index != kInvalidChunkIndex);
     const auto sourceFrameIndexRange = frameIndexRange(pAudioSource);
-    mixxx::AudioSourceStereoProxy audioSourceProxy(
-            pAudioSource,
-            tempOutputBuffer);
-    DEBUG_ASSERT(
-            audioSourceProxy.getSignalInfo().getChannelCount() ==
-            kChannels);
-    m_bufferedSampleFrames =
-            audioSourceProxy.readSampleFrames(
-                    mixxx::WritableSampleFrames(
-                            sourceFrameIndexRange,
-                            mixxx::SampleBuffer::WritableSlice(m_sampleBuffer)));
+
+    if (pAudioSource->getSignalInfo().getChannelCount() %
+                    mixxx::audio::ChannelCount::stereo() !=
+            0) {
```

Comment:
Whatever I try, it looks like clang format revert it back to that. Are we okay to just keep it as it? I guess I could put the `0 !=` at the beginning otherwise.

Reply:
Ok, I though it was easy.